app/src/main/java/com/vultisig/wallet/ui/navigation/NavController.kt

```
@@ -11,8 +11,12 @@ internal fun NavController.route(route: String, opts: NavigationOptions? = null)
     if (route == Destination.Back.route) {
         popBackStack()
     } else {
-        navigate(route) {
-            buildOptions(this, opts)
+      try {
+            navigate(route) {
+                buildOptions(this, opts)
+            }
+        } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)